src/types/index.d.ts

```diff
@@ -315,6 +315,34 @@ declare module CuriefenseClient {
 
   // Document other - END
 
+  // Integrations - START
+  
+  type RoleActionPair = {
+    idp_group?: string
+    reblaze_groups?: string[]
+  }
+
+  type Sso = {
+    enabled: boolean
+    title: string
+    provider: 'azure' | 'okta'
+    protocol: 'oauth2' | 'saml2'
+    roles_claim_key?: ''
```

Comment:
The type should be a string, not ''

```suggestion
    roles_claim_key?: string
```

Reply:
Done